README.md

```diff
@@ -43,6 +43,7 @@ C:\Code\NuKeeper\NuKeeper>dotnet run mode=organisation github_token=<GitToken> g
 | Name                             | Alias | Default value          | Required |
 |----------------------------------|-------|------------------------|-----------------|
 | mode                             | m     |                        | Yes             |
+| github_user                      | u     |                        | Yes             |
```

Comment:
Need to also add an explanation of parameter below and add the parameter to the examples above